### ISSUE TITLE
OpenVPN server certificate

### ISSUE THREAD

**Issue by lnovara** [sighupio/furyagent#31]
Furyagent is currently unable to renew the OpenVPN server certificate when it expires. This means that the user needs to resort to manual actions to renew this certificates.

**Comment by lzecca78** [comment]
Yes, this will help a lot the day by day activities and is a very nice `good first issue`.
I guess that the `culprit`  is the function `CreateUser` that should take the parameter `overwrite` in order to force the recreation/upload of the certificate with the same name.
